[Optimize] [PageConfig] Allow pipelineSchedulerConfig to be set from external json

usczz opened the pull request:
## Summary

Previously, pipelineSchedulerConfig was only included as a static template page configuration. Due to this limitation, different templates had to be compiled to for performance verification, which was quite inconvenient.

This pull request allows pipelineSchedulerConfig to be configured from external json source.

## Checklist

<!--- Check and mark with an "x" -->

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/lynx-family/lynx?pullRequest=129) <br/>All committers have signed the CLA.

usczz committed:
[Optimize] [PageConfig] Allow pipelineSchedulerConfig to be set from external json source

Previously, pipelineSchedulerConfig was only included as a static template page configuration.
Due to this limitation, different templates had to be compiled to for performance verification, which was quite inconvenient.

This pull request allows pipelineSchedulerConfig to be configured from external json source.

usczz commented:
/land

jianliang00 commented:
🛬 Landing process started! >>228.0

ShouruiSong commented:
/land

jianliang00 commented:
🛬 Landing process started! >>229.0
